tests/libres_tests/res/util/test_subprocess.py

```diff
@@ -65,6 +67,7 @@ def test_await_process_finished_tee(self):
             fh.write(bytearray(os.urandom(_maxBytes)))
 
         with open("a", "wb") as a_fh, open("b", "wb") as b_fh:
+            # pylint: disable=consider-using-with
```

Comment:
Same question as above: does `Does the comment ` # await_process_tee() ensures the process is terminated.` fit here?` fit here?

Reply:
Not on the scope, but nice names in this region :D